src/OrbitBase/include/OrbitBase/File.h

```diff
@@ -87,6 +90,9 @@ ErrorMessageOr<void> WriteFully(const unique_fd& fd, std::string_view content);
 // used for non-blocking reads from sockets/pipes - it does not handle EAGAIN.
 ErrorMessageOr<size_t> ReadFully(const unique_fd& fd, void* buffer, size_t size);
 
+ErrorMessageOr<size_t> ReadFullyAtOffset(const unique_fd& fd, void* buffer, size_t size,
```

Comment:
same disclaimer about `don't use for non-blocking reads` as for ReadFully applies here (maybe add a comment about that)

Reply:
done